lib-es5/utils/index.js

```diff
@@ -1063,6 +1063,8 @@ function process_request_params(params, options) {
   if (options.unsigned != null && options.unsigned) {
     params = exports.clear_blank(params);
     delete params.timestamp;
+  } else if (options.signature ) {
+    params = exports.clear_blank(options);
```

Comment:
Can you please explain what clear_blank is being used?

Reply:
Following the same practice the same as from line#1064